AmazonChimeSDK/AmazonChimeSDK.xcodeproj/project.pbxproj

```diff
@@ -173,7 +172,6 @@
 		00C9DC6A23FA393600876B41 /* DefaultVideoTileController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DefaultVideoTileController.swift; sourceTree = "<group>"; };
 		00D9DBB1241AE23F00146467 /* DefaultVideoClient.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DefaultVideoClient.swift; sourceTree = "<group>"; };
 		00EB5F602400591800007D3E /* VideoTile.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = VideoTile.swift; sourceTree = "<group>"; };
-		1C38DDCF24EC0DA4005524AB /* DeviceControllerTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DeviceControllerTests.swift; sourceTree = "<group>"; };
 		1C38DDD124EC0DBA005524AB /* MockVideoClientController.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = MockVideoClientController.swift; sourceTree = "<group>"; };
```

Comment:
Should this be removed as well?

Reply:
Removed. See the other comment for more detail.